### PR TITLE
fix(hx-pill): invert dismissible behaviour with persist

### PR DESCRIPTION
**Pull request by cathysiller**
JIRA: surf-1236

### LGTM's
- [x] Dev LGTM
- [x] Zoom LGTM

### CODE REVIEW

**Comment by HelixUI** [comment]
Deploy preview for *helix-ui* processing.

Built with commit 923083584dd14d1987e5fc95a2f3a6a1ef5f3f29

https://app.netlify.com/sites/helix-ui/deploys/5c004e02e5cd167f89851179

File: src/helix-ui/elements/HXPillElement.js [hunk 1]
```diff
@@ -32,12 +32,35 @@ export class HXPillElement extends HXElement {
...
+     *
+     * @default false
+     * @type {Boolean}
+     */
+    get persist () {
+        return this.hasAttribute('persist');
+    }
+    set persist (value) {
+        if (value) {
+            this.setAttribute('persist', ''); // boolean
```
Comment (100stacks) [inline src/helix-ui/elements/HXPillElement.js]: please remove trailing comment

File: src/helix-ui/elements/HXPillElement.less [hunk 2]
```diff
@@ -33,8 +33,10 @@
   }
 }
 
-:host([dismissable]) {
-  #hxDismiss {
-    display: inline-flex;
+:host([persist]) {
+  #hxPill {
```
Comment (100stacks) [inline src/helix-ui/elements/HXPillElement.less]: please remove the `#hxPill` wrapper

File: docs/elements/hx-pill/index.html [hunk 3]
```diff
@@ -51,16 +51,16 @@ <h2 id="methods">Methods</h2>
 
 {% block attributes %}
   <dl>
-    <dt>dismissable {Boolean [false]} <i>(optional)</i></dt>
+    <dt>persist {Boolean [false]} <i>(optional)</i></dt>
```
Comment (100stacks) [inline docs/elements/hx-pill/index.html]: * please remove the default value
* please remove the `(optional)` value

File: docs/elements/hx-pill/index.html [hunk 4]
```diff
@@ -51,18 +51,18 @@ <h2 id="methods">Methods</h2>
 
 {% block attributes %}
   <dl>
-    <dt>dismissable {Boolean [false]} <i>(optional)</i></dt>
+    <dt>persist {Boolean}</dt>
     <dd>
-      When present, pill will be dismissable.
+      When present, pill will be not dismissable.
```
Comment (100stacks) [inline docs/elements/hx-pill/index.html]: suggest changing this to: `When present, pill will not be dismissable.`

Comment (cathysiller) [reply docs/elements/hx-pill/index.html]: good catch, updated!!

**Pull request merged**